### PR TITLE
Auto-deploy releases to Github

### PR DESCRIPTION
**Pull request by diranged**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
-  - "pypy"
+- '2.7'
+- pypy
 install:
-  - make build
+- make build
 script: make test
 sudo: false
+before_deploy:
+  - make kingpin.zip
```
Comment (siminm) [inline .travis.yml]: Why spaces here but not in `python` or `install`?

**Comment by diranged** [comment]
@siminm, sweet check this out!

https://github.com/Nextdoor/kingpin/releases/tag/v0.3.1a
image

**Comment by diranged** [comment]
```(.venv)Matts-MacBook:kingpin diranged$ curl -L -O https://github.com/Nextdoor/kingpin/releases/download/v0.3.1a/kingpin.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   588    0   588    0     0   1172      0 --:--:-- --:--:-- --:--:--  1171
100 3007k  100 3007k    0     0  1057k      0  0:00:02  0:00:02 --:--:-- 1866
(.venv)Matts-MacBook:kingpin diranged$ python kingpin.zip  --version
0.3.0
```

(note the version string is wrong because i arbitrarily made up the tag)

**Comment by siminm** [comment]
That's pretty sweet :-D If damn boto didn't use a json file for their regions then we could actually use the zip for our actors.

**Pull request merged**